rancher/dashboard
# Fixed broken banner monitoring-receiver-ssl

### Summary
Fixes #8998

Fixed broken banner monitoring-receiver-ssl


### How to test

Go to  Cluster > Alerting > Routes and Receivers > Create new > select Email

Should have SSL banner fixed
